debian/xenial/kubelet/lib/systemd/system/kubelet.service

```diff
@@ -1,10 +1,12 @@
 [Unit]
-Description=Kubernetes Kubelet Server
-Documentation=https://github.com/kubernetes/kubernetes
+Description=kubelet: The Kubernetes Node Agent
+Documentation=http://kubernetes.io/docs/
```

Comment:
Nit: This should link to the kubelet doc page  http://kubernetes.io/docs/admin/kubelet/